tests/conftest.py

```diff
@@ -27,7 +27,10 @@
 
 
 def get_backend(backend_name):
-    if "-" in backend_name:
+    if "qiboml" in backend_name:
+        _, name = backend_name.split("-")
+        platform = None
+    elif "-" in backend_name:
         name, platform = backend_name.split("-")
```

Comment:
```suggestion
    if "-" in backend_name:
        name, platform = backend_name.split("-")
        name = platform if "qiboml" in backend_name else name
```

Reply:
I agree with this doubt. 
My decision was in the end to keep the old interface, but it generates an inconsistency with Qibojit indeed. 
My point was that it would be more natural to refer to e.g. `tensorflow` as a backend, like it is `numpy`, rather than a platform. 
But I am completely open to different povs.